DI5K03/FileManager/Commands/ClearAllCommand.cs

```diff
@@ -0,0 +1,33 @@
+using Filemanager.Infrastructure;
+using Filemanager.Model;
+
+namespace Filemanager.Commands
+{
+    internal class ClearAllCommand : ICommand
+    {
+        public string Name => "clear-all";
+
+        public async Task ExecuteAsync(IHost host, string[] args, Cache cache)
+        {
+            if (cache.Target_dir.Equals(""))
```

Comment:
üres string-nek string.Empty. Kifejezőbb, mint a `""`. Mégpedig azért, mert ez akár lehet véletlen eredménye is, míg az előbbi kifejezi a szándékot egyértelműen.